modules/nw-ingress-route-secret-load-external-cert.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/secured-routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-route-secret-load-external-cert_{context}"]
+= Creating a route with externally managed certificate
+
+:FeatureName: Securing route with external certificates in TLS secrets
+include::snippets/technology-preview.adoc[]
+
+You can configure {product-title} routes with third-party certificate management solutions by using the `.spec.tls.externalCertificate` field of the route API. You can reference externally managed TLS certificates via secrets, eliminating the need for manual certificate management. Using the externally managed certificate reduces errors and provides faster updates. 
```

Comment:
I would agree with @lunarwhite to reword this a bit. https://github.com/openshift/openshift-docs/pull/76392#issuecomment-2163492232